apps/www/components/ActionBar/AudioInfo.js

```diff
@@ -30,8 +39,56 @@ const AudioInfo = ({
   play,
   speakers = [],
   willBeReadAloud,
+  documentId,
+  documentPath,
 }) => {
   const [colorScheme] = useColorContext()
+  const { data, loading } = useDocumentSubscriptionQuery({
+    variables: {
+      path: documentPath,
+      onlyMe: true,
+    },
+    skip: !willBeReadAloud,
+  })
+  const [subscribe] = useSubscribeDocumentMutation()
+  const [unsubscribe] = useUnsubscribeDocumentMutation()
+
+  const readAloudSubscription = data?.document?.subscriptions?.nodes.find(
+    ({ object: { id } }) => id === documentId,
```

Comment:
Found out while testing that for articles which aren't a format and don't have a republik author profile, there are no nodes returned here.
E.g.:
https://www.republik.ch/2023/01/17/kunstkolumne-vahland-die-kunst-der-einigung
([graphql query](https://api.republik.ch/graphiql/?query=%7B%0A%20%20document(path%3A%20%22%2F2023%2F01%2F17%2Fkunstkolumne-vahland-die-kunst-der-einigung%22)%20%7B%0A%20%20%20%20id%0A%20%20%20%20subscriptions%3A%20subscribedBy(onlyMe%3A%20true)%20%7B%0A%20%20%20%20%20%20nodes%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20isEligibleForNotifications%0A%20%20%20%20%20%20%20%20filters%0A%20%20%20%20%20%20%20%20active%0A%20%20%20%20%20%20%20%20object%20%7B%0A%20%20%20%20%20%20%20%20%20%20...%20on%20Document%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A))

This means the subscribe button never comes up in such articles, for which the condition is:
`(!!data || !loading) && readAloudSubscription`

Probably this isn't wanted